access/types/forms.py

```diff
@@ -753,3 +773,20 @@ def get_randomized_checkbox_attributes(self, i, config, initial, correct,
         random_attributes['checksum'] = self.samples_hash(random_attributes['sample'])
         return selected_choices, correct_choices, initial_choices, random_attributes
 
+
+def subdiff(value, all_solutions):
+    solutions = all_solutions.split('|')
+    if len(solutions) > 1:
+        matching_parts = ["Multiple correct answers accepted."]
+    else:
+        matching_parts = []
+    for solution in solutions:
+        parts = "Correct parts in your answer: "
```

Comment:
Strings visible to the user must be translated (Finnish/English) -> Django `python manage.py makemessages` etc.